range.hpp

```diff
@@ -43,8 +43,8 @@ struct range_iter_base : std::iterator<std::input_iterator_tag, T> {
 
 template <typename T>
 struct range_proxy {
-    struct iter : detail::range_iter_base<T> {
-        iter(T current) : detail::range_iter_base<T>(current) { }
```

Comment:
Please fix the indentation here to be consistent with the rest (4 spaces)

Reply:
done!